source/Octopus.Client.Tests/PublicSurfaceAreaFixture.ThePublicSurfaceAreaShouldNotRegress..NETCore.approved.txt

```diff
@@ -3,7 +3,7 @@ Octopus.Client
   class DefaultLinkResolver
     Octopus.Client.ILinkResolver
   {
-    .ctor(Uri, String)
+    .ctor(Uri, String = "/api")

```

Comment:
I thought we could improve on this assent test by generating which properties are optional. I've only added optional properties so technically I didn't regress the public surface area but it's not clear from the required change to this test.

Reply:
I reverted this because it was a PITA to manage the changes...